src/otx/core/metrics/accuracy.py

```diff
@@ -303,7 +305,7 @@ def _apply(self, fn: Callable, exclude_state: Sequence[str] = "") -> nn.Module:
             )
             for acc in self.multiclass_head_accuracy
         ]
-        if self.num_multilabel_classes > 0:
+        if hasattr(self, "multilabel_accuracy"):
```

Comment:
How about using `self.multilabel_accuracy == None` instead of it?

Reply:
Is it okay to ask what the reason for this is? Is that a better way?

Reply:
Almost same but a little bit better thing to use it is that `self.multilabel_accuracy == None` is more explicit so it can deal with a case that attribute 'num_multilabel_classes' is set my mistake. If you want to leave it, you can ignore this comment :)

Reply:
i'll update it. you meant `self.multilabel_accuracy is None` right?

Reply:
Thanks. yes and also need to add `self.multilabel_accuracy = None` in the `__init__()`.

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/4042/commits/f055880f5c8ce01d604e1922e93f8563c3a0f904